Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -332,6 +332,8 @@ - (void)configureForThread:(TSThread *)thread
     _composeOnOpen = keyboardOnViewAppearing;
     _callOnOpen = callOnViewAppearing;
 
+    [self.uiDatabaseConnection beginLongLivedReadTransaction];
```

Comment:
We should update `uiDatabaseConnection` to the latest YapDatabase checkpoint _before_ we `ensureDynamicInteractions` to ensure that it operates on the latest contents of the thread.

The second `beginLongLivedReadTransaction` afterward will ensure that `uiDatabaseConnection` is up-to-date If `ensureDynamicInteractions` performs any mutations.